datasets/beheerkaart/cbs_grid/dataset.json

```
@@ -28,93 +28,243 @@
+          "dominantGgwGebied": {
             "type": "string",
-            "description": "Naam van gebiedsgerichtwerken gebied die overlapt met het BKT-CBS-GRID-100-Vlak."
+            "description": "Naam van dominant gebiedsgerichtwerken-gebied dat overlapt met het BKT-CBS-grid-100-vlak."
           },
           "dominantGgwGebiedcode": {
             "type": "string",
-            "description": "Code van dominant gebiedsgerichtwerken gebied die overlapt met het BKT-CBS-GRID-100-Vlak."
+            "description": "Code van dominant gebiedsgerichtwerken-gebied dat overlapt met het BKT-CBS-grid-100-vlak."
           },
           "percentageBlauw": {
             "type": "number",
-            "description": "Percentage kleur blauw (bgt_waterdeel) binnen het BKT-CBS-GRID-100-Vlak"
+            "minimum": 0,
+            "description": "Percentage kleur blauw (bgt_waterdeel) binnen het BKT-CBS-grid-100-vlak."
           },
           "percentageGrijs": {
             "type": "number",
-            "description": "Percentage kleur grijs (bgt_wegdeel,bgt_ondersteunend_wegdeel) binnen het BKT-CBS-GRID-100-Vlak."
+            "minimum": 0,
+            "description": "Percentage kleur grijs (bgt_wegdeel, bgt_ondersteunend_wegdeel) binnen het BKT-CBS-grid-100-vlak."
+          },
+          "percentageGroen": {
+            "type": "number",
+            "minimum": 0,
+            "description": "Percentage kleur groen (bgt_begroeid_terrein) binnen het BKT-CBS-grid-100-vlak."
+          },
+          "percentageOverig": {
+            "type": "number",
+            "minimum": 0,
+            "description": "Percentage overig (overige bgt_objectklassen) binnen het BKT-CBS-grid-100-vlak."
+          },
+          "percentagePubliekeRuimte": {
+            "type": "number",
+            "minimum": 0,
+            "description": "Totaal percentage publieke ruimte (>=10%) binnen het BKT-CBS-grid-100-vlak."
+          }
+        }
+      }
+    },
+    {
+      "id": "raster_10",
+      "type": "table",
+      "provenance": "bkt_beheerkaart_cbs_grid_raster_10",
+      "description": "Tabel met BKT-CBS-grid-raster (vlakken 10x10m2) en te beheren percentages publieke ruimte (totaal>=1%) op basis van de beheerkaart publieke ruimte.",
+      "version": "0.0.1",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "mainGeometry": "cbsGeometrie_10",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "id"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "provenance": "bk_cbs_grid_10_asd",
+            "type": "string",
+            "description": "Business-key: unieke aanduiding record (BKT-CBS-grid-10-vlak)."
+          },
+          "bkCbsGrid_100_asd": {
```

```suggestion
          "bkCbsGrid100Asd": {
```

Dit is de enige variant die onveranderd blijft bij conversie naar snake-case en terug naar camel-case.